logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
@guyboertje I'd vote no on putting any kind of guideline like this out then. It's just such a trivial improvement to the bytecode size only.
+ I think here and in all other cases where this kind of thing matters, the move to Java is incoming anyways, so fixing these spots for at best minor gains will not get us much imo.